### PR TITLE
[VP] Add Commercial Vehicles categories

### PR DESCRIPTION
**Pull request by ornanatan**
Add Commercial Vehicles category (Buses, Taxis, etc)

### CODE REVIEW

**Review by tylerrowsell — APPROVED** [review]

File: data/attributes.yml [hunk 1]
```diff
@@ -32352,6 +32352,31 @@ base_attributes:
...
+  values:
+  - bus_design_type__articulated
+  - bus_design_type__bi_articulated
+  - bus_design_type__double_decker
+  - bus_design_type__low_floor
+  - bus_design_type__minicoach
+  - bus_design_type__open_top
+  - bus_design_type__single_decker
+  - bus_design_type__step_entrance
+  - bus_design_type__other
```
Comment (ricardotejedorsanz) [inline data/attributes.yml:32369]: I would drop this attribute, as this brings multiple overlaps with the categories suggested: minicoach, double decker, articulated...

File: data/categories/vp_vehicles_parts.yml [hunk 2]
```diff
@@ -4828,3 +4829,170 @@
   - upholstery_pattern
   - watercraft_features
   - yacht_type
+- id: vp-2-4
+  name: Commercial Vehicles
+  children:
+  - vp-2-4-1
+  - vp-2-4-2
+  - vp-2-4-3
+  - vp-2-4-4
```
Comment (ricardotejedorsanz) [inline data/categories/vp_vehicles_parts.yml:4838]: For consistency, I would place these categories one level below, under `Vehicles & Parts > Vehicles > Motor Vehicles`

File: data/categories/vp_vehicles_parts.yml [hunk 3]
```diff
@@ -4828,3 +4829,170 @@
...
+  - upholstery_pattern
+- id: vp-2-4-2-2
+  name: Short Haul Semi-Trucks
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - bus_design_type
+  - fuel_supply
+  - item_condition
```
Comment (ricardotejedorsanz) [inline data/categories/vp_vehicles_parts.yml:4975]: We already have categories for `Trucks`, I wouldn't create a new area for them?

File: data/categories/vp_vehicles_parts.yml [hunk 4]
```diff
@@ -4828,3 +4829,170 @@
...
+- id: vp-2-4-4
+  name: Delivery Vans
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
```
Comment (ricardotejedorsanz) [inline data/categories/vp_vehicles_parts.yml:4998]: Same for Vans: we already have categories for `Vans` (electric, non-electric and hybrid).
I advise against creating a separate area to keep categories mutually exclusive.

**Review by ricardotejedorsanz — CHANGES_REQUESTED** [review]
I suggest to simplify the structure to avoid overlaps with existing categories.

**Pull request merged**